lms/djangoapps/email_marketing/tasks.py

```diff
@@ -17,6 +17,24 @@
 SAILTHRU_LIST_CACHE_KEY = "email.marketing.cache"
 
 
+@task(bind=True)
+def get_email_cookies_via_sailthru(self, post_parms):
+    try:
+        sailthru_client = SailthruClient(email_config.sailthru_key, email_config.sailthru_secret)
+        log.info(
+            'Sending to Sailthru the user interest cookie [%s] for user [%s]',
+            post_parms.get('cookies', ''),
+            user.email
+        )
+
+        sailthru_response = sailthru_client.api_post("user", post_parms)
+
+    except SailthruClientError:
```

Comment:
Why catch and throw again? If there isn't anything to do here, you don't even need to catch the exception